Program.cs

```diff
@@ -3,17 +3,52 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to swap the indices of a given list of numbers
+// A random series of numbers is displayed to the user and the user is asked to provide the two indices to be swapped.
+// The number series after swapping the indices is displayed.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Swap Indices</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method prints a series of numbers based on the count given by the user and displays the updated series after swapping the user asked indices</summary>
+      static void Main () {
+         Console.Write ("Enter the count(1-10) of random numbers to be displayed: ");
+
+         if (!int.TryParse (Console.ReadLine (), out int count)) Console.WriteLine ("Invalid Input.");
+         else {
+            int[] numberSeries = new int[count];
+            for (int i = 0; i < count; i++) {
+               Random random = new Random ();
+               numberSeries[i] = random.Next (1, 10);
+            }
+            foreach (int number in numberSeries) Console.Write (number + " ");
+
+            Console.Write ("\nEnter the indices to be swapped: \nFirst Index: ");
+
+            try {
+               if (!int.TryParse (Console.ReadLine (), out int firstIndex)) Console.WriteLine ("Invalid Input.");

```

Comment:
As there are multiple integer checks. instead of repeating, why not have a function to do that?